packages/nimble-components/src/combobox/specs/IxD.md

```diff
@@ -0,0 +1,238 @@
+# Nimble Combobox (IxD)
+
+## Overview
+
+The combobox component is used to present a pre-defined list of options that end-users can choose from or type to create a custom value. Comboboxes combine the flexibility of text input with the efficiency of selection for more user-friendly interactions.
+
+> THE VISUAL DESIGN IN THIS DOCUMENT MAY NOT BE ACCURATE
+
+### Background
+
+-   [Select and Combobox Refresh HLD (#1303)](https://github.com/ni/nimble/issues/1303)
+-   [ViD Combobox Styles (Figma)](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1529-41777&mode=design)
+
+## Usage
+
+When to use:
+
+-   To provide an input for text entry coupled with a popup for quick selection and filtering
+-   To allow end-users to enter custom values
+-   For entering settings or preference values that don't (yet) exist
+-   For single selection
+
+When not to use:
+
+-   When text entry is not needed for selection
```

Comment:
I'm not sure what _When text entry is not needed for selection_ means. Could you clarify? For example, does that mean when the user doesn't need to be able to enter a value that isn't in the pre-populated list?